packages/vsce/__tests__/__e2e__/wiremock/__files/CICSJVMEndpoint/records.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<response xmlns="http://www.ibm.com/xmlns/prod/CICS/smw2int" 
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xsi:schemaLocation="http://www.ibm.com/xmlns/prod/CICS/smw2int http://my-localhost:8080/CICSSystemManagement/schema/CICSSystemManagement.xsd" 
+version="3.0" connect_version="0610">
+	<resultsummary api_response1="1024" api_response2="0" api_response1_alt="OK" api_response2_alt="" recordcount="1" displayed_recordcount="1" cachetoken="E1812A8FA4625087" />
+	<records>
+		<cicsjvmendpoint _keydata="84858681A493A3C8A3A397C5958497968995A340404040404040404040404040404040404040404040404040" enablestatus="ENABLED" eyu_cicsname="MYREG1" eyu_cicsrel="E740" eyu_reserved="0" host="*" jvmendpoint="defaultHttpEndpoint" jvmserver="MYJVM1" port="28963" secport="N/A" type="com.ibm.ws.http" />
```

Comment:
All other mocked resources are named like this to make it REALLY obvious what they are. To the untrained eye, `defaultHttpEndpoint` does not tell you what it is.
```suggestion
		<cicsjvmendpoint _keydata="84858681A493A3C8A3A397C5958497968995A340404040404040404040404040404040404040404040404040" enablestatus="ENABLED" eyu_cicsname="MYREG1" eyu_cicsrel="E740" eyu_reserved="0" host="*" jvmendpoint="MYJVMENDPOINT1" jvmserver="MYJVM1" port="28963" secport="N/A" type="com.ibm.ws.http" />
```